commands/service_dev.go

```diff
@@ -62,7 +63,17 @@ func (c *serviceDevCmd) runE(cmd *cobra.Command, args []string) error {
 		printDeployStatuses(statuses)
 	}()
 
-	id, validationError, err := c.e.ServiceDeploy(c.path, statuses)
+	id, validationError, err := c.e.ServiceDeploy(c.path, nil, func(alias string) (bool, error) {
+		pretty.DestroySpinner()
+		var confirm bool
+		if err := survey.AskOne(&survey.Confirm{
+			Message: fmt.Sprintf("A service already exist with the same alias %q. Do you confirm to replace it?", alias),
```

Comment:
`A service already exist with the same alias %q. Do you confirm to replace it?`
to
`A service with the same alias %q already exists. Do you want to replace it?`

Reply:
done